tests/integration/testlib/examplelib.py

```diff
@@ -1,5 +1,5 @@
 #
```

Comment:
Proposal for the commit message:

```
tests, examplelib: Retry cleanup without verify

In case the cleanup example fails in the verification step, perform a second cleanup without verification to cleanup as much as possible.

Raise the cleanup exception to expose the error in the test since the cleanup examples should work as-is with verification.
```